aria/modeling/orchestration.py

```diff
@@ -327,20 +340,48 @@ class PluginBase(ModelMixin):
     uploaded_at = Column(DateTime, nullable=False, index=True)
     wheels = Column(List, nullable=False)
 
+    # region foreign keys
+
+    __private_fields__ = ['service_template_fk']
```

Comment:
as we mentioned, this should change; need two separate plugin models, one for those declared in service templates, and another for those installed via the plugin manager.